install/helm/agones/templates/controller.yaml

```diff
@@ -110,6 +110,18 @@ spec:
         # maximum port that can be exposed to GameServer traffic
         - name: MAX_PORT
           value: {{ .Values.gameservers.maxPort | quote }}
+        {{- if .Values.gameservers.portRanges }}
```

Comment:
Please reject `.Values.gameservers.portRanges` unless the gate is enabled. You can see an example of both feature gate detection and helm failure in this blob (which is no longer in the codebase - I think we've promoted all of the helm-relevant feature gates): https://github.com/googleforgames/agones/blob/3d57eaa4fe18590634c17137e369302c981b91b9/install/helm/agones/templates/controller.yaml#L15-L22